linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -83,7 +83,18 @@ private void run(String bomCoordinates)
     ImmutableSet<SymbolProblem> problemsInSnapshot =
         LinkageChecker.create(snapshot).findSymbolProblems().keySet();
 
+    Set<SymbolProblem> fixedErrors = Sets.difference(problemsInBaseline, problemsInSnapshot);
     Set<SymbolProblem> newErrors = Sets.difference(problemsInSnapshot, problemsInBaseline);
+
+    if (newErrors.equals(fixedErrors)) {
```

Comment:
These cannot be equal unless they're both empty. I think you mean `problemsInSnapshot.equals(problemsInBaseline)`. In that case you can move the `fixedErrors` and `newErrors` declarations to after this `if` block.